status-im/status-react
# Add flashlight functionality to choose recipient screen

### Summary:
Implement flashlight toggle on choose recipient screen

### Steps to test:
- Navigate to choose recipient (actual device, simulators can't test flashlight), toggle flashlight.
- Icon state should toggle, flashlight should turn on and off

![screen-one](https://user-images.githubusercontent.com/640347/30925589-c14c998e-a388-11e7-9c40-5b5f76e6085a.png)
![image-two](https://user-images.githubusercontent.com/640347/30925678-0b1248b6-a389-11e7-94d3-768d68f2ee35.png)


status: ready
